mailpile/plugins/keylookup/__init__.py

```diff
@@ -513,51 +518,92 @@ class KeyserverLookupHandler(LookupHandler):
     NAME = "PGP Keyservers"
     LOCAL = False
     TIMEOUT = 20  # We know these are slow...
-    PRIVACY_FRIENDLY = False
+    PRIVACY_FRIENDLY = True
     PRIORITY = 200
     SCORE = 1
 
+    # During testing, there were frequent HTTP gateway errors returned from
+    # hkps.pool.sks-keyservers.net so sks-keyservers.net was added too.
+    KEY_SERVER_BASE_URLS = [
+        "https://sks-keyservers.net/pks/lookup",
+        "https://hkps.pool.sks-keyservers.net/pks/lookup",
+    ]
```

Comment:
In my experience https://keyserver.ubuntu.com/ has the highest uptime and lowest response times.